docs/references/pysdk_api_reference.md

```diff
@@ -103,25 +103,60 @@ infinity_obj.disconnect()
 Infinity.create_database(db_name, conflict_type = ConflictType.Error)
 ```
 
-Creates a database with a given name. `conflict_type` specifies the policy when a database with the same name exists. 
+Creates a database with a specified name.
 
 ### Parameters
 
-- **db_name : str(not empty)** Name of the database.
-- **conflict_type : ConflictType** `enum` The countermeasure to take when a database with the same name exists.  See `ConflictType`, which is defined in the **infinity.common** package: 
-  - `Error`
-  - `Ignore`
+#### db_name: `str`, *Required*
+
+Name of the database. Must not be empty. 
+
+#### conflict_type: `ConflictType`, *Optional*
+
+Conflict policy in `enum` for handling situations where a database with the same name exists. 
+
+  - `Error`: Raise an error if a database with the same name exists.
+  - `Ignore`: Ignore the table creation requrest and keep the database with the same name as-is.
+  - `Replace`: Drop the existing database and create a new one. 
```

Comment:
remove

Reply:
what about `replace` in `create_table()`?